src/main.rs

```diff
@@ -61,23 +61,55 @@ fn main() {
     let force = matches.is_present("force");
     let bench = matches.is_present("bench");
     let open = matches.is_present("open");
-    let config = match Config::new(app_package, verbose, quiet, force, bench, open) {
+
+    let mut config = match Config::new(app_package, verbose, quiet, force, bench, open) {
         Ok(c) => c,
         Err(e) => {
             print_warning(format!("There was an error when reading the config.toml file: {}",
                                   e),
                           verbose);
-            let mut c: Config = Default::default();
-            c.set_app_package(app_package);
-            c.set_verbose(verbose);
-            c.set_quiet(quiet);
-            c.set_force(force);
-            c.set_bench(bench);
-            c.set_open(open);
-            c
+            Config::default()
         }
     };
 
+    config.set_app_package(app_package);
+    config.set_verbose(verbose);
+    config.set_quiet(quiet);
+    config.set_force(force);
+    config.set_bench(bench);
+    config.set_open(open);
+
+    if let Some(threads) = matches.value_of("threads") {
+        config.set_threads(threads.parse().expect("An integer"));
```

Comment:
`parse()` ensures this, however in a not very userfriendly way. How should I instead do this?